sandpack-client/src/types.ts

```diff
@@ -379,4 +379,5 @@ export type Template =
   | "static"
   | "solid"
   | "nextjs"
-  | "node";
+  | "node"
+  | "static";
```

Comment:
For what it's worth, "static" is already listed as an option on line 379 so this change to the `Template` type is probably not needed 😄 Exciting to see the static template being brought into Sandpack - massively appreciate your work on this!